packages/locale/src/index.ts

```diff
@@ -1 +1,2 @@
 export { default as TranslatedString } from './TranslatedString/TranslatedString';
+export { LocaleContext, LocaleContextType, useLocale } from './context';
```

Comment:
`contexts` if you intend for this folder to contain all the context components. Otherwise, name it after the component itself, e.g.: `LocaleContext`

Reply:
I think that's the idea I was going for, to have a contexts folder. I will update that accordingly 😄 . Thanks